client/test/unit/specs/transactions.spec.js

```diff
@@ -4,58 +4,14 @@ import VDragged from 'v-dragged'
 import Transactions from '@/components/application/Transactions'
 import store from '@/store'
 import { cleanState } from '../../features/specs/support/clean.state'
-import sinon from 'sinon'
-import axios from '@/axios-auth'
```

Comment:
I just went straight to STORE with the values for these unit tests as i was more concerned about testing the template and component logic vs mocked requests. 
The requests are part of the component for sure, however it is just a means to populate the store with data.
If you want to expand these tests to populate the store via the requests, please do :)

Reply:
I agree with your decision.

I will add tests for the header and transaction items in a future commit...